src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 span:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
DRY (don't repeat yourself), especially with this line
```
background-image: url(images/star-active.svg);
```

it could have been implemented in a more simple way?
```
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background: url("./images/star-active.svg") center no-repeat;
}
```